popup/warehouse.js

```diff
@@ -55,27 +59,58 @@ function createList(storeObj) {
 				// });
 			});
 
+			accButton.addEventListener("keydown" , event => {
+				if (accButton.textContent.length > 30) {
+					accButton.textContent = accButton.textContent.substring(0, 30);
+				} else {
+					if (accButton.textContent.length === 0) {
+						accButton.textContent =  obj.title;
+					}
+				}
+			});
+
+			accButton.addEventListener("focusout", () => {
```

Comment:
Here the `focusout` event works as intended on Firefox. However on Chrome things got weird.  

From my testing done on Chrome this is what I think is happening, when a click is done outside the extension, the `focusout` event is not raised. The `focusout` event is raised only when the click is done outside the element listening but inside the Extension popup.  
I see only two options to make it work. If you have any idea of your own, please let me know,
1. Apart from checking the `focusout` event  also check another event with an OR condition. The other event could be `onPointerLeave`. More about it can be found [here](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onpointerleave)
2. Save the text when an `Enter` or `Carriage Return` is received. For this to be the case though, there needs to be some visual to indicate that the title is not yet saved and when it is saved and possibly a tooltip which says "Hit Enter to confirm edit". This is so that people aren't very tech-savy can also easily use the feature  

Again, I'd like to hear your thoughts on this. Is there any alternate elegant way to know when to save it?

Reply:
`pointerleave` seems is working as well, except when you are not using the mouse, even with the OR condition. I will keep working on this, since I want to keep this as simple as possible.

Reply:
I  did another round last night, i load the extension in Chrome as well and it seems it is working fine on both browsers whit this block:

```
['pointerleave', 'focusout'].forEach( function(e) {
	accButton.addEventListener(e, () => {
		if(obj.title != accButton.textContent){
			obj.title = accButton.textContent;
			persist(id, obj.title, obj.tabs, obj.tags);
		}
	});
});
```
So, no additional action from the user for saving the text would be required

what do you think?